app/uk/gov/hmrc/nisp/assets/sass/nisp.scss

```diff
@@ -357,9 +359,21 @@ h1.title-spa {
   padding-bottom: 0px;
 }
 
-table.cope-table p{   
+table.cope-table td {
```

Comment:
I prefer styles that are not too specific. I would prefer just the class i.e .cope-table